css/catalog.scss

```diff
@@ -32,3 +32,7 @@
   margin-top: 3%;
   margin-bottom: 3%;
 }
+
+footer.footer {
```

Comment:
You mean you're using a redundant class name to be able to increase the specificity of the selector? Are you sure your CSS inclusion order is proper?